pulsar-io/debezium/mysql/src/main/java/org/apache/pulsar/io/debezium/mysql/DebeziumMysqlSource.java

```diff
@@ -0,0 +1,109 @@
+    static private final String DEFAULT_TENANT = "public";
+    static private final String DEFAULT_NAMESPACE = "default";
+    static private final String DEFAULT_OFFSET_TOPIC = "debezium-mysql-offset-topic";
+    static private final String DEFAULT_HISTORY_TOPIC = "debezium-mysql-history-topic";
+
+    private static void throwExceptionIfConfigNotMatch(Map<String, Object> config,
+                                                       String key,
+                                                       String value) throws IllegalArgumentException {
+        Object orig = config.get(key);
+        if (orig == null) {
+            config.put(key, value);
+            return;
+        }
+
+        // throw exception if value not match
+        if (!orig.equals(value)) {
+            throw new IllegalArgumentException("Expected " + value + " but has " + orig);
+        }
+    }
+
+    private static void setConfigIfNull(Map<String, Object> config, String key, String value) {
+        Object orig = config.get(key);
+        if (orig == null) {
+            config.put(key, value);
+        }
+    }
+
+    // namespace: tenant/namespace/
+    private static String topicNamePrefix(SourceContext sourceContext) {
+        String tenant = sourceContext.getTenant();
+        String namespace = sourceContext.getNamespace();
+
+        return ((tenant == null || tenant.isEmpty()) ? DEFAULT_TENANT : tenant) + "/" +
+            ((namespace == null || namespace.isEmpty()) ? DEFAULT_NAMESPACE : namespace) + "/";
+    }
+
+    @Override
+    public void open(Map<String, Object> config, SourceContext sourceContext) throws Exception {
+        // connector task
+        throwExceptionIfConfigNotMatch(config, TaskConfig.TASK_CLASS_CONFIG, DEFAULT_TASK);
+
+        // key.converter
+        setConfigIfNull(config, PulsarKafkaWorkerConfig.KEY_CONVERTER_CLASS_CONFIG, DEFAULT_CONVERTER);
+        // value.converter
+        setConfigIfNull(config, PulsarKafkaWorkerConfig.VALUE_CONVERTER_CLASS_CONFIG, DEFAULT_CONVERTER);
+
+        // database.history implementation class
+        setConfigIfNull(config, MySqlConnectorConfig.DATABASE_HISTORY.name(), DEFAULT_HISTORY);
+
+        // database.history.pulsar.service.url, this is set as the value of pulsar.service.url if null.
+        String serviceUrl = (String) config.get(PulsarKafkaWorkerConfig.PULSAR_SERVICE_URL_CONFIG);
+        if (serviceUrl == null) {
+            throw new IllegalArgumentException("Pulsar service URL not provided.");
+        }
+        setConfigIfNull(config, PulsarDatabaseHistory.SERVICE_URL.name(), serviceUrl);
+
+        String topicNamePrefix = topicNamePrefix(sourceContext);
+        // topic.namespace
+        setConfigIfNull(config, PulsarKafkaWorkerConfig.TOPIC_NAMESPACE_CONFIG, topicNamePrefix);
+
+        // database.history.pulsar.topic: history topic name
+        setConfigIfNull(config, PulsarDatabaseHistory.TOPIC.name(),
+            topicNamePrefix + DEFAULT_HISTORY_TOPIC);
```

Comment:
I think you should use `<tenant>/<namespace>/<connector-name>-<DEFAULT_HISTORY_TOPIC>`. 

Image following use case, I have 2 database servers and I submitted 2 connectors. If people doesn't set the history topic name, these 2 connectors would eventually share the same history topic, which can cause corruption.